src/Factory.php

```diff
@@ -112,11 +128,19 @@ function($value) {
             return $proxy;
         }
 
-        return $proxy->save()->withoutAutoRefresh(function(Proxy $proxy) use ($attributes) {
+        $proxy->save();
+
+        $proxy->withoutAutoRefresh(function(Proxy $proxy) use ($attributes) {
```

Comment:
```suggestion
        $proxy->save()->withoutAutoRefresh(function(Proxy $proxy) use ($attributes) {
```

Reply:
This don't solve the problem

Reply:
I didn't think it would, this was just a minor thing.